.gitignore

```diff
@@ -7,6 +7,7 @@ _build
 .project
 .pycharm_helpers/
 .pydevproject
+.vscode/
```

Comment:
Hi! I recommend that we don't add more editor-specific configurations in .gitignore.

Check out your own personal gitignore! https://tekin.co.uk/2020/03/maintain-a-global-git-ignore-file

Reply:
Is this a policy? As a matter of best practice I disagree (I think `.gitignore` files that avoid developer fatfinger risk are always good practice), but if this is org-wide policy I certainly will follow it.

Reply:
It's something that I've been asked to do before, I think by @feanil

Reply:
I think it's fine to add `.vscode` as it's a standard IDE home especially given the other ones we have here.  I would be more concerned if it was something that was developer specific such that it would not be useful for anyone or might lead to confusion about why a file isn't getting tracked.

Reply:
Fine with me. I've been stopped from adding it to this repo before.